### DISCUSSION TITLE
Unable to Restore Any of the Paused Projects due to an error

### DISCUSSION THREAD

**Discussion by neil-rubens** [supabase/supabase#2643] [Questions] [answered]
When I try to restore **any** of the paused projects under my account; I get the following error:

`Unexpected token C in JSON at position 0Unexpected token C in JSON at position 0`

and the db never gets restored; please assist.

Projects:

https://app.supabase.io/project/dewvmkigyuhepsujkwoi/building
https://app.supabase.io/project/pvxvadntlnyclynsdcdj/building

**Comment by dragarcia** [comment, marked as answer]
Hi @neil-rubens 👋 

We're taking a look at it now!

**Reply by neil-rubens** [reply]
Thank you 😃

**Reply by dragarcia** [reply]
We've taken note of this bug and are now looking at how to squash it. 

In the meantime, we've unpaused the two projects you've mentioned and they should be in the midst of being restored now. Do give it a while for the restoration to take place!

**Reply by dragarcia** [reply]
Both projects should be restored now 😃

**Reply by neil-rubens** [reply]
@daiangan thank you for such a quick response! 
Db is working now.

**Reply by shared2023** [reply]
> Hi @neil-rubens 👋
> 
> We're taking a look at it now!

Hi, our project got the a same problem and it was be paused. The restore button cannot be clickable and we cannot restart it, either. Could you help us to solve it? Thank you!
